resources/compute_instance_out.go

```diff
@@ -0,0 +1,349 @@
+package resources
+
+import (
+	"encoding/json"
+	"fmt"
+	conv "github.com/googleinterns/terraform-cost-estimation/memconverter"
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jedib0t/go-pretty/v6/text"
+	"strings"
+)
+
+// The number of columns in output table for ComputeInstance.
+const numColumns = 5
+
+// ToTable creates a table.Table and fills it with the pricing information from
+// ComputeInstanceState.
+func (state *ComputeInstanceState) ToTable() (*table.Table, error) {
+	if state == nil {
+		return nil, nil
+	}
```

Comment:
Can state == nil in practice? It's better if the caller doesn't call this method on a nil ref.